catalog/MDCDragons/HeaderView.swift

```diff
@@ -45,15 +45,7 @@ class HeaderView: UIView {
                                      fillColor: .white)
     imageView.image = image
 
-    // To make the search icon tinted white we have to reach the internal UITextField of the UISearchBar
-    if let searchBarTextField = self.searchBar.value(forKey: "_searchField") as? UITextField,
```

Comment:
you just need to remove the underscore from `_searchField` -> `searchField`

Reply:
Why was all this code needed before if the tintColor works just fine on iOS 9+?

Reply:
I believe it didn’t work well at the time. If you are able to set the tint color and by that set the magnifying glass image then please open a PR and we can close this one.

Reply:
This PR fixes a crasher. Let’s discuss how to polish this separately please.

Reply:
If there's no visual change, this is fine. It's just surprising that we had such complex code when we could have just set the tint color instead.

Reply:
Please remove the underscore, it fixes the crash.

Reply:
Otherwise, there will be visual changes afaik.

Reply:
If this changes the rendering of something, please include before and after screenshots.  Since iOS 13 is in Beta and this is for Dragons, we need not rush this through.